backend/src/packages/chaiNNer_pytorch/pytorch/io/save_model.py

```diff
@@ -44,14 +44,14 @@ class WeightFormat(Enum):
     side_effects=True,
 )
 def save_model_node(
-    model: PyTorchModel, directory: str, name: str, weight_format: WeightFormat
+    model: ModelDescriptor, directory: str, name: str, weight_format: WeightFormat
 ) -> None:
     full_file = f"{name}.{weight_format.value}"
     full_path = os.path.join(directory, full_file)
     logger.debug(f"Writing model to path: {full_path}")
     if weight_format == WeightFormat.PTH:
-        torch.save(model.state_dict(), full_path)
+        torch.save(model.state_dict, full_path)
```

Comment:
Now that I see this, why do we even store a state dict in the descriptor? Couldn't we just do `desc.model.state_dict()`? Aren't we just wasting RAM by always having the original state dict around?

Reply:
Because I completely forgot that you could access the state_dict this way... Yeah we should probably get rid of that